tests/integration/test_dcim.py

```diff
@@ -0,0 +1,139 @@
+import pytest
+
+
+class TestSimpleServerRackingAndConnecting:
```

Comment:
Instead of setting up class based tests where we would have to do lots of spin up repeatedly we could also use something like the pytest-depends module to just spin up some commonly used objects and reuse them a lot.. but that has its own caveats as well. Maybe a mix of the two is best?